packages/frontend/src/store.ts

```diff
@@ -507,6 +507,10 @@ export const defaultStore = markRaw(new Storage('base', {
 		where: 'device',
 		default: { type: 'syuilo/bubble2', volume: 1 } as SoundStore,
 	},
+	mutedReactions: {
+		where: 'account',
+		default: [],
```

Comment:
他と同じように as を付けてあげた方が良いと思う
```suggestion
		default: [] as string[],
```